README.md

```diff
@@ -327,9 +325,29 @@ const draft = nylas.drafts.build({
   replyToMessageId: MESSAGE_ID,
 });
 
+// Enabling tracking
+
+  const draft = nylas.drafts.build({
```

Comment:
np: since we have a code snippet above that calls `const draft = nylas.drafts.build({`let's define a tracking constant here instead of another draft. Then it will be clearer below what `tracking` is when we pass it into `send()`